src/i18n/locales/en-US.json

```diff
@@ -118,6 +118,9 @@
   "cmps.designer.link.Backend.version": "Version",
   "cmps.designer.link.Backend.status": "Status",
   "cmps.designer.link.ChangeBackendButton.btnText": "Change Backend",
+  "cmps.designer.link.ChangeTaroBackendButton.btnText": "Change Taro Backend",
+  "cmps.designer.link.ChangeTaroBackendButton.error": "Error",
```

Comment:
Remove because this isn't used by the ChangeTaroBackendButton component

Reply:
Please see my prior comments about changes to this file. They haven't been updated in your latest commits.